bowling/src/ArgumentParser.cpp

```diff
@@ -0,0 +1,34 @@
+#include "ArgumentParser.hpp"
+
+#include <algorithm>
+
+ArgumentParser::ArgumentParser(int argc, char** argv) {
+    argc_ = argc;
+    argv_.reserve(argc_);
+    for (int i = 0; i < argc_ ; ++i) {
+        argv_.push_back(argv[i]);
+    }
+    appName_ = argv_[0];
+}
+
+std::string ArgumentParser::getInputDirectory() const {
+    if (argc_ == 1) {
+        return "";
+    }
+    return argv_[1]; 
```

Comment:
Practicly you return first argument if exists,  in `getOutputFileName` you are returning secound argument if exists.  
It's good to make it into small function like this:

```cpp
std::string getExistingArgument(size_t number) {
    if (size >= number)
        return arguments_[number - 1];
    return ""; 
}

what do you think?
```